content/networks-metadata.json

```diff
@@ -0,0 +1,2324 @@
+[
```

Comment:
this file should not be here, it is gitignored. So remove and commit the removal should remove it from PR too

Reply:
this file still needs to be removed